### ISSUE TITLE
Duplicate DBS IDs

### ISSUE THREAD

**Issue by TobiasNx** [hbz/lobid-organisations#480]
7 | AG292
5 | RP560
2 | AB026
2 | BD624
2 | BW221
2 | CM653
2 | CQ873
2 | CU131
2 | CX954
2 | DA580
2 | DA701
2 | DA798
2 | DB869
2 | DC564
2 | DK868
2 | DP096
2 | EJ882
2 | GA523
2 | GS182
2 | GX708
2 | HR716
2 | JM285
2 | JQ584
2 | JX665
2 | JX999
2 | LH102
2 | LJ003
2 | PR403
2 | SB551
2 | WD730
2 | WD961
2 | WE205
2 | WE594
2 | WK720
2 | WN435

**Comment by TobiasNx** [comment]
We should check if we change #462 if all duplicates are still there.

**Comment by TobiasNx** [comment]
@acka47 should we contact for DBS errors the DBS team?

**Comment by acka47** [comment]
> should we contact for DBS errors the DBS team?

Yes, please, collect them first (maybe in the dienst-wiki?) and we will then send them to DBS team.

**Comment by TobiasNx** [comment]
@acka47 has send these errors to DBS.

**Comment by katjajana** [comment]
Hey, the ISIL are no duplicates but the DBS-Id entries seem incorrect. I need to check with DBS and ask them which ones to keep.
